dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckerTest.java

```diff
@@ -42,13 +48,30 @@
       "testdata/proto-google-cloud-firestore-v1beta1-0.28.0.jar";
   private final String EXAMPLE_CLASS_FILE =
       "testdata/grpc-google-cloud-firestore-v1beta1-0.28.0_FirestoreGrpc.class";
+  private final List<Path> firestoreDependencies = Arrays.asList(
```

Comment:
This is a great case for `ImmutableList`. And because it's known to be immutable, you can name it with ALL_CAPS as a constant.

```
private static final ImmutableList<Path> FIRESTORE_DEPENDENCIES =
    ImmutableList.of(
        absolutePathOfResource(...),
        absolutePathOfResource(...),
        absolutePathOfResource(...));
````